draft-ietf-quic-multipath.md

```diff
@@ -183,15 +208,6 @@ capitals, as shown here.
 We assume that the reader is familiar with the terminology used in
 {{QUIC-TRANSPORT}}. When this document uses the term "path", it refers
 to the notion of "network path" used in {{QUIC-TRANSPORT}}.
```

Comment:
The path ID is a new concept which differs from RFC9000. I think it's better to update the definition of path and path id, but not to delete all this definition of the new concept.

Reply:
it's already covered in the intro now.